queries/sql/highlights.scm

```diff
@@ -0,0 +1,105 @@
+(string) @string
+(number) @number
+(comment) @comment
+
+(function_call
+    function: (identifier) @function)
+
+(identifier) @variable
+
+[
+  (NULL)
+  (TRUE)
+  (FALSE)
+] @constant.builtin
```

Comment:
Maybe true/false even boolean , but can also understand them as built-in constants in database land

Reply:
Is there a list of classes somewhere? I probably used constant because it's defined as constant in [tree-sitter-python](https://github.com/tree-sitter/tree-sitter-python/blob/78c4e9b6b2f08e1be23b541ffced47b15e2972ad/queries/highlights.scm#L37-L41)

Reply:
The allowed captures are listed in CONTRIBUTING.MD